### ISSUE TITLE
Copy-mode on mouse movement

### ISSUE THREAD

**Issue by Thomas485** [pvolok/mprocs#91]
Hello,

I have the problem, that, if i clicked in the terminal window mprocs is running in, the copy-mode is activated by mouse-movement.
Is this the intended behavior?

After Copying the selection or exiting the copy-mode any mouse-movement would start it again.
I only observe this behavior in mprocs, everything else is working as expected (leftclick-drag or rightclick on start and end points to select text)

My current workaround is to left click trice and right click once. This somehow disables the copy-mode until I clicked in the window again.

I'm using `rxvt-unicode (urxvt) v9.30 - released: 2021-11-27` and `GNU bash, Version 5.2.15(1)-release (x86_64-pc-linux-gnu)` to call mprocs.

**Comment by liskin** [comment]
Weirdly, this also happens when mprocs is run in tmux 🤯

**Comment by liskin** [comment]
Right, turns out this is actually a bug in urxvt: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1059148

Fixed upstream in https://github.com/exg/rxvt-unicode/commit/5ec6fb5ec7f0871f19814279575a2f7873e5556a (unfortunately not in any release yet).
